DESCRIPTION

```diff
@@ -37,6 +37,11 @@ Imports:
     rtables,
     ggplot2.utils,
     hermes,
-    teal.modules.hermes (>= 0.1.5.9009)
+    teal.modules.hermes (>= 0.1.5.9009),
+    goshawk,
+    teal.goshawk,
+    osprey,
+    teal.osprey,
+    nestcolor
```

Comment:
ok that this fixes the problem, but seems this is just a workaround that should ultimately fixed in `staged.dependencies` e.g.? because in the BAC we don't use any of these packages directly

Reply:
The problem is that the workflow which generates the stable version of Biomarker Catalog doesn't use `staged.dependencies` for dependency installation.

Instead, it uses `devtools::install_dev_deps()` to get the dependencies from https://insightsengineering.r-universe.dev and CRAN.